### ISSUE TITLE
Setup errors are reported as config errors

### ISSUE THREAD

**Issue by matthew-ia** [matthew-ia/cayo#40]
This is because the config validation checks any path-related values, including the defaults. So if the default is not found (user setup error), then it reports it as a config error. The message could just be changed to be more easily understood if it's a default value that's causing the error.

**Comment by matthew-ia** [comment]
Might just handle this altogether with the new validation error handling (fe42faa); surely there's a way to know if the user passed a value vs the default. Then:
``` 
if (userValue) report error as Config error
else report as Setup error
```

**Comment by matthew-ia** [comment]
Fixed in 606dc8a2b8c7ac829cc0b3e49b949f569cb07f3d
